worker_tasks/scraper.py

```diff
@@ -30,9 +32,15 @@ def scrape_articles(categorizer):
 
         # Extract text and categorize item
         text_only = soup.get_text()
-        item.category_1, item.category_2 = categorizer.categorize_item(item.title, text_only)
+        categories = categorizer.categorize_item(item.title, text_only)
+        if len(categories) >= 1:
+            item.category_1 = categories[0]
+
+        if len(categories) >= 2:
+            item.category_2 = categories[1]
```

Comment:
So when we have a category table going, this will just be iterating over ``categories`` and adding CategoryItems to the category_item table?